requirements.txt

```diff
@@ -5,3 +5,5 @@ wheel
 pytest
 pytest-cov
 mock
+numpy
+scipy
```

Comment:
Numpy and scipy should already be dependencies within the setup.py script so in theory they shouldn't be required here.